getting_started/5_Quantum_Teleportation.ipynb

```diff
@@ -0,0 +1,285 @@
+    }
+   },
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The Quantum Teleportation Protocol \n",
```

Comment:
I think it would be really helpful if you could also describe why this protocol works. Thanks!